### PR TITLE
feat(docs): FRMW-171 - Migrate rxCompile docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-171

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

**Comment by 100stacks** [comment]
LGTM

File: src/rxCompile/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [![stable](http://badges.github.io/stability-badges/dist/stable.svg)](http://github.com/badges/stability-badges)
 
-Allows compilation of expressions inside of other expressions.
\ No newline at end of file
+A component allows compilation of expressions inside of other expressions.
```
Comment (CITguy) [inline src/rxCompile/README.md]: A component **that** allows compilation ...

File: src/rxCompile/rxCompile.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * @description
  * # rxCompile Component
  *
- * [TBD]
+ * A component allows compilation of expressions inside of other expressions.
```
Comment (CITguy) [inline src/rxCompile/rxCompile.js]: Same as above.

File: src/rxCompile/README.md (hunk 1 above)
Comment (bry4n) [inline src/rxCompile/README.md]: done

**Comment by CITguy** [comment]
LGTM

**Pull request merged**